apps/reader/src/components/viewlets/AnnotationView.tsx

```diff
@@ -51,8 +52,49 @@ const AnnotationPane: React.FC = () => {
     )
   }, [focusedBookTab?.book.annotations])
 
+
+  const exportAnnotations = () => {
+    const annotations = (focusedBookTab?.book.annotations ?? []).map((a) => ({ ...a }))
+    // process annotations to be under each section
+    // group annotations by title
+    const grouped = group(annotations, (a) => a.spine.title)
+    const exported: Record<string, any[]> = {}
+    for (const chapter in grouped) {
+      const annotations = grouped[chapter]?.map((a) => {
+        const annotation: Record<string, any> = {}
+        if (a.notes !== undefined) annotation.notes = a.notes
+        if (a.text !== undefined) annotation.text = a.text
+        return annotation
+      }) ?? []
+      exported[chapter] = annotations
+    }
+
+    // Copy to clipboard as markdown
+    const exportedAnnotationsMd = Object.entries(exported).map(([chapter, annotations]) => {
+      return `## ${chapter}\n${annotations.map((a) => `- ${a.text} ${a.notes ? `(${a.notes})` : ''}`).join('\n')}`
+    }).join('\n\n')
+    navigator.clipboard.writeText(exportedAnnotationsMd)
+
+    // Download as markdown
+    const blob = new Blob([exportedAnnotationsMd], { type: 'text/markdown' })
```

Comment:
Feel free to remove this download as markdown download blob

Reply:
To make it simple, can we just copy it as markdown?

Reply:
Done! just removed the download as markdown snippet

Reply:
@pacexy thank you for the feedback! excited to see this copy to clipboard as markdown on the site :)